Hardcode system user UID according to the MassOS 'USERIDS.md' specification.

Also set up the home directory of the system user, as it is unset by default
but needed by MassOS.

--- a/system-helper/flatpak.conf.in
+++ b/system-helper/flatpak.conf.in
@@ -1 +1 @@
-u @SYSTEM_HELPER_USER@ - "Flatpak system helper" -
+u @SYSTEM_HELPER_USER@ 963 "Flatpak system helper" /var/lib/flatpak
